### DISCUSSION TITLE
Unable to upload images to PRs.

### DISCUSSION THREAD

**Discussion by malaybaldha-ta** [community/community#146275] [Pull Requests]
### Select Topic Area

Question

### Body

Unable to upload of images or any other files in PRs, "uploading <file>" message flashes but no file gets uploaded. Is it a known issue?

**Comment by queenofcorgis** [comment]
Hey there!  👋 

Thanks for posting in the GitHub Community, @malaybaldha-ta ! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category. The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them.

I've gone ahead and moved this to the correct category for you. Good luck!

**Comment by ghostinhershell** [comment]
Hey guys, thanks so much for reaching out in our community! It looks like there was an incident reported on our Status page yesterday that affected both Pull Requests and API Requests: https://www.githubstatus.com/incidents/w6g0cmvyx3vm. This issue was resolved, shortly after it first appeared, yesterday at 20:05 UTC.

For future reference, you can check our [GitHub Status Page](https://www.githubstatus.com/) to see for real-time and historical data on system performance + subscribe to issues for updates.

Cheers!

**Reply by malaybaldha-ta** [reply]
Hi @ghostinhershell, thankyou for reaching out, however I am still **unable** to upload media in PRs, is there any way to know why this behavior is isolated for me?

**Reply by santhoshramgk-ta** [reply]
> Hey guys, thanks so much for reaching out in our community! It looks like there was an incident reported on our Status page yesterday that affected both Pull Requests and API Requests: https://www.githubstatus.com/incidents/w6g0cmvyx3vm. This issue was resolved, shortly after it first appeared, yesterday at 20:05 UTC.
> 
> For future reference, you can check our [GitHub Status Page](https://www.githubstatus.com/) to see for real-time and historical data on system performance + subscribe to issues for updates.
> 
> Cheers!

Hello @ghostinhershell, even I am facing this issue, no multimedia file is being uploaded in PRs, issues, or even this community comment, kindly look into it once again.
